lisa/sut_orchestrator/azure/tools.py

```diff
@@ -101,6 +97,13 @@ def get_resource_disk_mount_point(self) -> str:
         waagent_configuration = self.get_configuration()
         return waagent_configuration["ResourceDisk.MountPoint"]
 
+    def is_autoupdate_enabled(self) -> bool:
+        waagent_configuration = self.get_configuration()
+        if waagent_configuration.get("AutoUpdate.Enabled") == "n":
+            return False
+        else:
+            return True  # if set or not present, defaults to "y"
```

Comment:
Good comment to help understanding the default! Don't put comment behide the code. Put it on top of code.